README.md

```diff
@@ -10,6 +10,76 @@ It can be used by installing this package & setting the `extends` property in yo
 }
 ```
 
+Alternative yml config  `.eslintrc.yml`:
```

Comment:
Are we using YAML config anywhere? If so, I think it's easy enough to guess from the provided JSON without adding a lot of extra noise to the readme.